### ISSUE TITLE
Point defense calculation error

### ISSUE THREAD

**Issue by jsomara** [herosheets/woin-starships#53]
The point defense bonus to defense needs to be divided by the ship class.  Also, that modified value is a conditional value- against fighters and missiles only – so it shouldn’t add directly to the base DEFENSE score.  Rather, better note it under Point Defenses as (DEFENSE +x vs. fighters and missiles)

**Comment by seiyria** [comment]
So as a base, for example, Railgun is +2 defense. If you have a class I ship, it's +2 defense still. Class 2 is +1 defense, and class 10 is +.2 defense, but it should show as +(modified value) vs fighters and missiles (for ALL defenses)?

**Comment by Morrus** [comment]
I can’t quite parse that sentence, but here’s how it works:

Railgun is +2 DEFENSE.  If you have 10 railguns, it’s +20 DEFENSE.  But you divide that by Ship Class, so on a Class 5 ship, that’s 20/5 = +4 DEFENSE.

But it only applies in certain situations, so it shouldn’t modifiy the actual defense score, just be noted in parenthesis in the point defense entry. 

For example:

Point Defenses 50x Extel Harrington Systems BB-4 heavy turbolaser battery (DEFENSE vs missiles & fighters +5)

**Comment by seiyria** [comment]
Okay, that makes more sense.

**Comment by Morrus** [comment]
![point_defenses](https://cloud.githubusercontent.com/assets/14054539/10320584/1d34152a-6c6c-11e5-8fe0-a0ba3348acb2.jpg)

Just looked at the changes made.  It’s still wrong I’m afraid.

You appear to have divided the initial DEFENSE by the class and put that in parenthesis.

Let me try again (I've attached an image, if that helps)

The DEFENSE score is unaffected.

The point defenses score is divided by class.  The original DEFENSE is not.  

That value (point defenses/3) is what goes in parenthesis in the format “(+x vs. fighters and missiles).  

This only needs to show in the point defenses line, not the DEFENSE line.

**Comment by seiyria** [comment]
That strikes me as odd: as anyone using this I would want to know what contributes what and to what they contribute as (ie, I'd want to see both totals and both values so I can see where they're coming from), but I digress. I will make this change shortly.

**Comment by Morrus** [comment]
Awesome!  It’s now correct – thanks!  It needs a “+” sign in front of it, and to be rounded to an integer, but it works!

It’s still adding to the DEFENSE score though. The DEFENSE score is not modified by this.

**Comment by herosheets-codeship** [comment]
Closed with commit from today, defense not modified by point defnese
